fdbclient/NativeAPI.actor.cpp

```diff
@@ -2784,13 +2785,14 @@ void updateTagMappings(Database cx, const GetKeyServerLocationsReply& reply) {
 ACTOR Future<KeyRangeLocationInfo> getKeyLocation_internal(Database cx,
                                                            Optional<TenantName> tenant,
                                                            Key key,
-                                                           SpanID spanID,
+                                                           SpanContext spanContext,
                                                            Optional<UID> debugID,
                                                            UseProvisionalProxies useProvisionalProxies,
                                                            Reverse isBackward,
                                                            Version version) {
 
-	state Span span("NAPI:getKeyLocation"_loc, spanID);
+	// TODO - ljoswiak. Parent or Link?
+	state Span span("NAPI:getKeyLocation"_loc, spanContext);
```

Comment:
This looks right as is. The span for this function has a parent which is the span from the calling function.